fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -162,15 +150,14 @@ void writeResponse(final FcrepoResponse response, final File file)
 
         final List<URI> describedby = response.getLinkHeaders("describedby");
         for (final Iterator<URI> it = describedby.iterator(); describedby != null && it.hasNext(); ) {
```

Comment:
@ajs6f, yes, this looks like a potential NPE situation. Maybe a simple `if describedby != null` check would do the trick?